src/main/java/edu/berkeley/ground/db/CassandraClient.java

```diff
@@ -21,12 +21,7 @@
 import com.datastax.driver.core.ResultSet;
 import com.datastax.driver.core.Session;
 
-import edu.berkeley.ground.model.versions.GroundType;
-
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
I believe we're using [Google's Java Style Guide](https://google.github.io/styleguide/javaguide.html), which states that "Wildcard imports, static or otherwise, are not used."